azure/firewall/variables.tf

```diff
@@ -74,33 +74,20 @@ variable "firewall_dns_servers" {
   default     = null
 }
 
+variable "private_ip_ranges" {
+  description = "A list of SNAT private CIDR IP ranges, or the special string IANAPrivateRanges, which indicates Azure Firewall does not SNAT when the destination IP address is a private range per IANA RFC 1918."
```

Comment:
Perhaps this can be validated somehow?

Reply:
added CIDR validation

Reply:
Ahh, sorry, I wasn't clear.

I meant validate the input so that it only accepts a list of RFC1918 addresses?

EG:
https://stackoverflow.com/questions/73860207/terraform-regex-how-to-validate-ipv4-or-private-cidr-as-per-the-rfc1918

It's more of a nice to have. I'll approve the change anyway, but if you've got time, it makes sense to add this in before merging.

Reply:
created an issue to look at this later